Bottom: a619f9fba3ef9f36439d4535646efde5faaec137
Top:    848cb7193ac44dbfeb19f412c13d9c1df926c1e4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 15:53:41 -0500

Refresh of gitlab-status-reporter

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 9b6a7d23..8aa9bb73 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -52,6 +52,7 @@ container_env:
   GITLAB_URI: 'https://{{ gitlab_hostname }}/'
   GITLAB_CLIENT_ID: '{{ gitlab_buildbot_auth_id }}'
   GITLAB_CLIENT_SECRET: '{{ gitlab_buildbot_auth_secret }}'
+  GITLAB_TOKEN: '{{ gitlab_buildbot_token }}'
   GITLAB_WEBHOOK_SECRET: '{{ buildbot_webhook_secret }}'
   LDAP_URI: 'ldap://{{ openldap_hostname }}:389/'
   LDAP_BIND_USER: 'cn=proxyagent,dc=novatech'
